src/App.tsx

```diff
@@ -1,8 +1,105 @@
-import React from 'react';
-import './App.css';
+import React, { useState } from 'react';
+import { TodosList } from './TodosList';
 
-const App = () => (
-  <h1>Dynamic list of TODOs</h1>
-);
+import { fullTaskList } from './api';
+import './App.scss';
+
+const App = () => {
+  const [todos, setTodos] = useState<FullTaskList>([]);
+  const [dataLoaded, setdataLoaded] = useState<boolean>(false);
+  const [loading, setLoading] = useState<boolean>(false);
+  const [button, switchButton] = useState<boolean>(true);
```

Comment:
this isn't an appropriate name for a boolean value
try `isButtonVisible`, `setButtonVisibility`